core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -1398,7 +1402,8 @@ object LogManager {
       logDirFailureChannel = logDirFailureChannel,
       time = time,
       keepPartitionMetadataFile = keepPartitionMetadataFile,
-      interBrokerProtocolVersion = config.interBrokerProtocolVersion)
+      interBrokerProtocolVersion = config.interBrokerProtocolVersion,
+      remoteStorageSystemEnable = config.getBoolean(RemoteLogManagerConfig.REMOTE_LOG_STORAGE_SYSTEM_ENABLE_PROP))
```

Comment:
Use `config.remoteLogManagerConfig.enableRemoteStorageSystem()` instead of checking the specific property in config as it is already built.

Reply:
Yes, it's better to use `onfig.remoteLogManagerConfig.enableRemoteStorageSystem()`. Updated.